client/src/components/tools.jsx

```diff
@@ -32,36 +33,36 @@ const Tools = () => {
     <div className="gm_tools">
       <div className="gm_tools-content">
         <Menu pointing>
-          <Menu.Item
-            disabled
-            active={selected === 'account'}
-            onClick={updateTab('account')}
-          >
-            User Management
-          </Menu.Item>
+          {' '}
           <Menu.Item
             active={selected === 'online'}
             onClick={updateTab('online')}
           >
             Who's Online
           </Menu.Item>
+          <Menu.Item
+            active={selected === 'account'}
+            onClick={updateTab('account')}
+          >
+            User Management
+          </Menu.Item>
           <Menu.Item active={selected === 'items'} onClick={updateTab('items')}>
             Item Search
           </Menu.Item>
           <Menu.Item active={selected === 'chars'} onClick={updateTab('chars')}>
             Player Search
-          </Menu.Item>
+          </Menu.Item>{' '}
           <Menu.Item active={selected === 'yells'} onClick={updateTab('yells')}>
             Yells
           </Menu.Item>
         </Menu>
-        {/* {selected === 'account' && <Accounts />} */}
         {selected === 'online' && <OnlineList />}
+        {selected === 'account' && <Accounts />}
         {(item || selected === 'items') && (
           <Itemsearch itemname={item} itemstack={stack} />
         )}
         {(player || selected === 'chars') && <Playersearch charname={player} />}
-        {selected === 'yells' && <YellTab />}
+        {yells || (selected === 'yells' && <YellTab />)}
```

Comment:
Remove the `yells ||` check again, since that query param is never set.

Reply:
I added the query param actually and returned the tools page to priority to Accounts as the opening tab.